eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -67,7 +73,7 @@ stages:
 
   jobs:
 
-  # PR and CI legs ------------------------------------
+  # dotnet-lite PR and CI legs and dotnet CI legs ------------------------------------
```

Comment:
Prior to this change there were comments that clearly delineated the leg sections by type (e.g. stage 1, stage 2, etc.)  This is no longer the case therefore I lean towards removing this comments as they are going to cause confusion.

Reply:
Removed in [this commit](https://github.com/dotnet/installer/pull/17370/commits/aad69ec5a52c6b74d4f6e4dfd4edc2ecdf8e115b)